src/routes/controllers/projects/createProject.js

```diff
@@ -49,6 +48,20 @@ const createProject = async (req, res) => {
             ];
         }
 
+        if (Array.isArray(links)) {
+            if (links.length) {
+                const { parameterPlaceholders, parameters } = verifyNewLinks(
```

Comment:
I realized the way I was doing it by inserting `parameterPlaceholders` was not the stable way of doing it, and there was an easier way. It's whatever, though; I took note of it and might write it in as potential bugs/fixes/improvements (that we won't do).